pkg/endpoint/endpoint.go

```diff
@@ -1592,19 +1576,23 @@ func (e UpdateStateChangeError) Error() string { return e.msg }
 // if there was an issue triggering policy updates for the given endpoint,
 // or if endpoint regeneration was unable to be triggered.
 func (e *Endpoint) Update(owner Owner, cfg *models.EndpointConfigurationSpec) error {
-	e.Mutex.Lock()
+	var lockerr error
```

Comment:
Why do use `lockerr` instead of the standard `err` convention?

Reply:
I started using it in bpf.go because `err` was used in several big functions in a convoluted way and after that I decided that it's nice to have a distinctly named error just for lock errors. But maybe it's too much. Can change it.

Reply:
Please change it. We will likely backport this and it should be as small as possible.

Reply:
Done